addons/mail/wizard/mail_resend_partner_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mail_resend_partner_view_form" model="ir.ui.view">
+            <field name="name">mail.resend.partner.view.form</field>
+            <field name="model">mail.resend.partner</field>
+            <field name="groups_id" eval="[Command.link(ref('base.group_user'))]"/>
+            <field name="arch" type="xml">
+                <form string="Partner">
+                    <header>
+                        <button name="action_resend" type="object" string="Resend" class="oe_highlight"
+                            attrs="{'invisible': [('failure_reason', 'in', (False, ''))]}"/>
+                    </header>
+                    <sheet>
+                        <div class="oe_button_box" name="button_box">
+                            <button class="oe_stat_button" name="action_open_record"  type="object" icon="fa-user">
+                                <span>Open <br/> Record</span>
+                            </button>
+                        </div>
+                        <group>
+                            <field name="notification_id" invisible="1"/>
+                            <field name="partner_readonly" invisible="1"/>
+                            <field name="name" readonly="1" required="1"/>
+                            <field name="email" attrs="{'readonly': [('partner_readonly', '=', True)]}" class="mw-50"/>
```

Comment:
"mw-50" could break form layout I think, can't you get the desired result by playing with multiple groups and/or colspan?